apps/casdoor/1.779.0/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   casdoor:
-    image: casbin/casdoor:v1.779.0
+    image: casbin/casdoor:v1.780.0
     container_name: ${CONTAINER_NAME}
     restart: always
     networks:
```

Comment:
There is one minor change between the two lines:

1. Changed Docker image version from "v1.779.0" to "v1.780.0".

This update indicates that there have been no major issues with the Casdoor service in this version compared to its predecessor.

Optimization Suggestion:
If you plan on using these containers in production, consider implementing automated deployment pipelines or using environment variables for setting default configurations instead of hardcoding them like `${CONTAINER_NAME}`. This approach makes it easier to manage and extend without modifying multiple configuration files each time an update requires it.

Other than this specific version change, the code appears correct and follows best practices for a Docker compose file.